src/rotator_library/error_handler.py

```
@@ -885,16 +1658,20 @@ def is_server_error(e: Exception) -> bool:
     """Checks if the exception is a temporary server-side error."""
     return isinstance(
         e,
-        (ServiceUnavailableError, APIConnectionError, InternalServerError, OpenAIError),
+        (ServiceUnavailableError, APIConnectionError, InternalServerError),
     )
 
 
 def is_unrecoverable_error(e: Exception) -> bool:
```

Ruff (0.15.4)

[warning] Unused function argument: `e`

(ARG001)